src/core_plugins/tagcloud/public/tag_cloud_visualization.js

```diff
@@ -67,6 +67,7 @@ export class TagCloudVisualization {
   }
 
   async render(data, status) {
+    if (!(status.resize || status.data || status.params)) return;
```

Comment:
nit: Can you write this with brackets?
IMHO I read it  better this as  
```
if (!status.resize && !status.data && !status.params) {
  return;
}
```
if there is no update on resize, and no update on data and no update on params than return.